src/plugins/platforms/eglfs/deviceintegration/eglfs_kms/qeglfskmsgbmscreen.cpp

```diff
@@ -75,6 +75,21 @@ static const char *formatGLError(GLenum err)
     }
 }
 
+static PFNGLEGLIMAGETARGETTEXTURE2DOESPROC imageTargetTexture2DProc = NULL;
```

Comment:
```suggestion
 static PFNGLEGLIMAGETARGETTEXTURE2DOESPROC imageTargetTexture2DProc = nullptr;
```

use `nullptr` instead